README.md

```diff
@@ -126,10 +126,6 @@ Click `File` > `New` > [...scroll down...] `Terminal` to launch a terminal in yo
 
 ## 6. Clone this GitHub Repo in the Terminal
 
-```
-cd ~/SageMaker && git clone https://github.com/data-science-on-aws/workshop
```

Comment:
The fact that it was duplicated was the typo. It looks like the day after I made this PR that the change was made ([commit](https://github.com/data-science-on-aws/workshop/commit/82ecc63d17b2db41877f2d2a52b1afac51aa88cf#diff-04c6e90faac2675aa89e2176d2eec7d8)) and since then you've added it back and made it clear it's a essential step not to miss :)